components/input/input.html

```diff
@@ -0,0 +1,192 @@
+          </li>
+          <li class="drawer-item">
+            <a href="/components/avatar/avatar.html">Avatar</a>
+          </li>
+          <li class="drawer-item">
+            <a href="/components/badge/badge.html">Badge</a>
+          </li>
+          <li class="drawer-item">
+            <a href="/components/button/button.html">Button</a>
+          </li>
+          <li class="drawer-item">
+            <a href="/components/card/card.html">Card</a>
+          </li>
+          <li class="drawer-item">
+            <a href="/components/image/image.html">Image</a>
+          </li>
+          <li class="drawer-item drawer-item-active">
+            <a href="/components/input/input.html">Input</a>
+          </li>
+          <li class="drawer-item"><a>Lists</a></li>
+          <li class="drawer-item"><a>Modal</a></li>
+          <li class="drawer-item"><a>Navigation</a></li>
+          <li class="drawer-item"><a>Rating</a></li>
+          <li class="drawer-item"><a>Grid</a></li>
+          <li class="drawer-item"><a>Slider</a></li>
+          <li class="drawer-item"><a>Toast (Snackbar)</a></li>
+        </ul>
+      </aside>
+
+      <main class="main">
+        <header class="header">
+          <div class="header-title">Input</div>
+          <div class="description">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Optio,
+            voluptas et a sapiente fugit doloribus.
+          </div>
+        </header>
+
+        <div class="divider"></div>
+
+        <div class="content">
+          <div class="content-title">Text Input with label</div>
+          <div class="description">
+            To create stylized text input, put your input label and input
+            element inside a div with class 'input-group'.
+          </div>
+          <div class="demo">
+            <div class="input-group">
+              <label for="username">Username</label>
+              <input
+                type="text"
+                id="username"
+                class="input"
+                placeholder="Enter your username"
+              />
+              <label for="password">Password</label>
+              <input
+                type="password"
+                id="password"
+                class="input"
+                placeholder="Enter your password"
+              />
+            </div>
+          </div>
+          <div class="demo-code">
+            <pre>
+                <code class="html">
+  &lt;div class="input-group"&gt;
+    &lt;label for="username"&gt;Username&lt;/label&gt;
+    &lt;input type="text" class="input" id="username" placeholder="Enter your username" /&gt;
+    &lt;label for="password"&gt;Password&lt;/label&gt;
+    &lt;input type="password" class="input" id="password" placeholder="Enter your password" /&gt;
+  &lt;/div&gt;
+                </code>
+            </pre>
+          </div>
+        </div>
+
+        <div class="divider"></div>
```

Comment:
you can use hr here

Reply:
Will have to fix that on every page, will do it.